src/Mod/OpenSCAD/prototype.py

```diff
@@ -661,7 +661,7 @@ def readfile(filename):
         f = pythonopen(tmpfile)
     else:
         f = pythonopen(filename)
-    rootnode =  p arsenode(f.read())[0]
```

Comment:
what's up with this? shouldn't this be it's own PR?

Reply:
I've used this [tool](https://github.com/bndr/pipreqs) to generate the list. It uses python's AST module to parse all the files it can find and filter out import statements. To do that all the files must have correct syntax or it won't work.
This is one of the files that had an error, since it's such a small thing I though it was fine to include it here. 

There were other files with errors: they are quite old and automatically generated, and I thought it was best not to touch them, so I excluded the directory they live in from the scan. There's a [test](https://github.com/FreeCAD/FreeCAD/blob/master/src/Mod/Test/TestPythonSyntax.py) which uses AST and works much the same way to test FreeCAD python files syntax. Notice how the files I just mentioned are whitelisted there.

Reply:
There's actually no need to keep that file around at all, so I suggest removing this change from this PR, and then make a PR that removes that file entirely.